config/routes.rb

```diff
@@ -8,6 +13,13 @@
   devise_scope :user do
     get '/users/sign_out' => 'devise/sessions#destroy'
   end
+  match 'users/:id' => 'users#show', via: :get
+  # or
+  get 'users/:id' => 'users#show'
+  get 'users' => 'users#index'
+  # or
+  resources :users, only: [:show]
+
```

Comment:
What are you trying to do here. Why did you declare multiple routes for same action?

Reply:
I just tried different ways of routing now I removed them.
Done

Reply:
Well. There is no restriction on try. But refactor and clean your code before publishing.